thinc/layers/reduce_first.py

```diff
@@ -17,19 +19,13 @@ def reduce_first() -> Model[Ragged, OutT]:
 def forward(
     model: Model[Ragged, OutT], Xr: Ragged, is_train: bool
 ) -> Tuple[OutT, Callable[[OutT], Ragged]]:
```

Comment:
We probably want to use `InT` here as well for consistency? And in the `forward` of `reduce_last`.

Reply:
Fixed.